app/models/accreditation_institute_campus.rb

```diff
@@ -0,0 +1,38 @@
+# frozen_string_literal: true
+
+class AccreditationInstituteCampus < ActiveRecord::Base
+  self.table_name = 'accreditation_institute_campuses'
```

Comment:
Could not get Rails to properly pluralize 'campus'.  (I did try adding inflections)